pymess/utils/migrations.py

```diff
@@ -0,0 +1,23 @@
+import codecs
+import os
+
+from chamber.shortcuts import change_and_save
+
+from pymess.config import settings
+
+
+class SyncEmailTemplate:
```

Comment:
Is this a temporary thing? I don't see any usage...

Reply:
migrations.RunPython(SyncEmailTemplate(slug)),

Reply:
this is correct usage and no this is not a temporary thing